tests/viscoelastic_fixed_elastic_time_step.prm

```diff
@@ -0,0 +1,159 @@
+# This test checks viscoelastic stress buildup with a
```

Comment:
If this is mostly a copy of an existing benchmark, it would make sense to just include the benchmark in this prm file instead of copying everything (see for example the [mid_ocean_ridge](https://github.com/geodynamics/aspect/blob/master/tests/mid_ocean_ridge.prm) test).

Reply:
This test does derive from the [viscoelastic stress build-up benchmark](https://github.com/naliboff/aspect/blob/06aa82b90890c03e60e63c80ac947194da534bcf/benchmarks/viscoelastic_stress_build-up/viscoelastic_stress_build-up.prm), but there are a number of differences and the other viscoelastic tests are variants from this test. 

My preference is to have one test parameter file with all of the parameters specified. However, I could derive the other two viscoelastic tests from viscoelastic_fixed_elastic_time_step.prm to clean things up.

Reply:
I think it's fine to have separate parameter files that specify all of the parameters. My only point was, if there is one parameter file that basically represents the benchmark case (maybe just with a lower resolution and shorter end time), it makes sense to include the benchmark prm file rather than copying it, as we will immediately notice when a pull request breaks that test (while the benchmarks are not run for every pull request), so the benchmark prm file will always be working and up to date.

Reply:
I actually used the benchmark case as a template for the test for exactly that reason, but having it derive directly from the benchmark would make that perfectly clear for anyone else who runs into issues. Definitely worth the change :+1:

Reply:
Yes, and the other big point is that in case we ever do changes to the structure of the parameter file again, we will notice if we have to update the benchmark parameter file (in the past, it happened that we just updated the test prm file, and forgot about the file in the benchmarks folder, as it wasn't used directly in any test).